src/main/java/filemanager/model/XmlProductModel.java

```diff
@@ -1,12 +1,21 @@
 package filemanager.model;
 
-import java.math.BigDecimal;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
 
+@JsonIgnoreProperties({"category", "quantity", "sku"})
```

Comment:
There is a better way to exclude all unknown properties during deserialization. Check out ```ignoreUnknown``` parameter of this annotation.